src/test/java/org/folio/search/repository/IndexRepositoryTest.java

```diff
@@ -78,7 +78,7 @@ void createIndex_negative_throwsException() throws IOException {
       .isInstanceOf(SearchServiceException.class)
       .hasCauseExactlyInstanceOf(IOException.class)
       .hasMessage("Failed to perform elasticsearch request "
-        + "[index=test-resource_test-tenant, type=createIndexApi, message: err]");
```

Comment:
what the reason of this change?

Reply:
The tenant name is used to create postgresql schema, but `test-tenant` is not a valid identifier, so I had to change it to `test_tenant`.

Reply:
This should be discussed, maybe I missed some information about tenant name requirements, it should be always in lower snake case?

Reply:
Case is not sensitive here, but it should be a valid postgres identifier I think.